### PR TITLE
Added ConnectionPool implementation

### PR DESCRIPTION
**Pull request by ElamanK**
Please let me know what needs to be changed, thanks!

### CODE REVIEW

File: src/main/java/threads/threads_practice/ThreadImplements.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        for (int i = 0; i < 6; i++) {
+            LOGGER.info(i);
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                LOGGER.info(e);
+            }
+        }
+    }
+}
```
Comment (ankochergin) [inline src/main/java/threads/threads_practice/ThreadImplements.java]: check formatting

File: src/main/java/threads/threads_practice/ThreadExtends.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            LOGGER.info(i);
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                LOGGER.info(e);
+            }
+        }
+        }
+    }
+
```
Comment (ankochergin) [inline src/main/java/threads/threads_practice/ThreadExtends.java]: formatting

**Review by ankochergin — CHANGES_REQUESTED** [review]
1. check formatting in all classes
2. remove IntelliJ IDEA files from git
3. remove target folder from git

**Comment by ElamanK** [comment]
removed idea files and fixed formatting issue

**Comment by ElamanK** [comment]
target folder removed

**Pull request merged**